src/main/java/com/openlattice/postgres/PostgresColumn.java

```diff
@@ -20,22 +20,11 @@
 
 package com.openlattice.postgres;
 
-import static com.openlattice.postgres.PostgresDatatype.BIGINT;
-import static com.openlattice.postgres.PostgresDatatype.BIGINT_ARRAY;
-import static com.openlattice.postgres.PostgresDatatype.BOOLEAN;
-import static com.openlattice.postgres.PostgresDatatype.BYTEA;
-import static com.openlattice.postgres.PostgresDatatype.INTEGER;
-import static com.openlattice.postgres.PostgresDatatype.INTEGER_ARRAY;
-import static com.openlattice.postgres.PostgresDatatype.JSONB;
-import static com.openlattice.postgres.PostgresDatatype.TEXT;
-import static com.openlattice.postgres.PostgresDatatype.TEXT_ARRAY;
-import static com.openlattice.postgres.PostgresDatatype.TIMESTAMPTZ;
-import static com.openlattice.postgres.PostgresDatatype.UUID;
-import static com.openlattice.postgres.PostgresDatatype.UUID_ARRAY;
-
 import com.openlattice.IdConstants;
 import com.openlattice.edm.type.Analyzer;
 
+import static com.openlattice.postgres.PostgresDatatype.*;
```

Comment:
🚫 
`import static .*` considered harmful. Unless this has changed in recent JVMs, I believe this will import every static anything in the class, even if unused. Admittedly PostgresDatatype is probably the only type of class where I might accept this but even so, personally I'd pref just listing everything used explicitly.

Reply:
👍